test/sass/logical.spec.scss

```diff
@@ -1,17 +1,17 @@
 @use "test" as *;
 @use "../../src/shared-styles/validation" with ( $is-test: true);
-@use "../../src/shared-styles/logical-properties";
+@use "../../src/shared-styles/logical";
```

Comment:
`logical` seems a reasonable filename / namespace, from https://caniuse.com/#search=logical it doesn't look like it would cause a conceptual clash with anything else. Allows for use as `logical.property` which reads well.